docs/source/markdown/options/volume.md

```diff
@@ -6,12 +6,16 @@
 
 Create a bind mount. If `-v /HOST-DIR:/CONTAINER-DIR` is specified, Podman
 bind mounts `/HOST-DIR` from the host into `/CONTAINER-DIR` in the Podman
-container. Similarly, `-v SOURCE-VOLUME:/CONTAINER-DIR` mounts the named
-volume from the host into the container. If no such named volume exists,
+container. When specifying a host path, the host path must exist. Podman will
+not create the host path and will return an error. File globbing is supported,
+--volume /HOST-DIR/foo\* as long as destination paths are not specified.
```

Comment:
I feel like this needs its own paragraph, it's sufficiently different from the rest (no destination supported) that it feels like it needs to be distinct.